<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DimensionForm</class>
 <widget class="QGroupBox" name="DimensionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dimensión</string>
  </property>
  <property name="title">
   <string>DIMENSIONES</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="loteAnchoLbl">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ancho Medio:</string>
     </property>
     <property name="buddy">
      <cstring>loteAnchoEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="loteAnchoEdit">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="loteLargoLbl">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Largo Medio:</string>
     </property>
     <property name="buddy">
      <cstring>loteLargoEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="loteLargoEdit">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="loteSuperficieLbl">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Superficie (A):</string>
     </property>
     <property name="buddy">
      <cstring>loteSuperficieEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="loteSuperficieEdit">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="loteOcupacionLbl">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ocupación (B/100)/A:</string>
     </property>
     <property name="buddy">
      <cstring>loteOcupacionEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="loteOcupacionEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="loteEdificabilidadLbl">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Edificabilidad (A)/(C):</string>
     </property>
     <property name="buddy">
      <cstring>loteEdificabilidadEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="loteEdificabilidadEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>cambiado()</slot>
 </slots>
</ui>
